patches/@deadcanaries+granax+3.2.5.patch

```diff
@@ -67,7 +48,72 @@ index edd3ca0..e606b93 100644
        default:
          throw new Error(`Unsupported platform "${platform}"`);
      }
-@@ -261,7 +262,7 @@ exports.install = function(callback) {
+@@ -136,9 +137,28 @@ exports._unpackMacintosh = function(bundle, callback) {
+     path.join(BIN_DIR, '.tbb.dmg')
+   ], { cwd: BIN_DIR });
+ 
++  const finalback = (err, value) => {
++    const extract = childProcess.spawn('hdiutil', [
++      'detach',
++      path.join(BIN_DIR, '.tbb')
++    ], { cwd: BIN_DIR });
++
++    extract.on('close', (code) => {
++      if (code < 0) {
++        const unpackErr = new Error('Failed to unpack bundle');
++        if (err) {
++          unpackErr = new AggregateError([unpackErr, err], 'Multiple errors attempting to unpack Tor');
++        }
++        return callback(unpackErr);
++      }
++
++      callback(null, value);
++    });
++  };
```

Comment:
A smaller patch would suffice but I took the liberty to make this clean up after itself along the unhappy path.